server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGates.java

```diff
@@ -47,11 +43,10 @@
 import static org.sonar.server.user.AbstractUserSession.insufficientPrivilegesException;
 import static org.sonar.server.util.Validation.CANT_BE_EMPTY_MESSAGE;
 import static org.sonar.server.util.Validation.IS_ALREADY_USED_MESSAGE;
-import static org.sonar.server.ws.WsUtils.checkFound;
 import static org.sonar.server.ws.WsUtils.checkRequest;
 
 /**
- * Methods from this class should be moved to {@link QualityGateUpdater} and to new classes QualityGateFinder / QualityGateConditionsUpdater / etc.
+ * Methods from this class should be moved to {@link QualityGateUpdater} and to classes QualityGateFinder / QualityGateConditionsUpdater / etc.
  * in order to have classes with clearer responsibilities and more easily testable (without having to use too much mocks)
  */
 public class QualityGates {
```

Comment:
Good point, we could indeed do it. The reason why it hasn't been done yet is because not all methods are moved elsewhere, it's a work-in-progress (and a big part will be done during our sprint!)